assets/react/v3/entries/course-builder/components/modals/AssignmentModal.tsx

```diff
@@ -115,6 +117,7 @@ const AssignmentModal = ({
       time_duration: {
         value: '0',
         time: 'weeks',
+        after_start: false,
```

Comment:
This settings should not be part of time_duration. Use the key in root of object. Key name suggestions
```
calculate_deadline_after_start
deadline_from_start
```